providers/base/bin/camera_quality_test.py

```diff
@@ -99,15 +126,7 @@ def get_score_from_device(device: str, save: bool = False) -> float:
         scores.append(score)
         iter_count += 1
 
-    # Save the image if requested
-    if save:
-        with NamedTemporaryFile(
-            prefix="camera_test_brisque_{}".format(device),
-            suffix=".jpg",
-            delete=False,
-        ) as f:
-            cv2.imwrite(f.name, image)
-            print("Saved image to {}".format(f.name))
+    save_image(image, device)
```

Comment:
If you use a `--output/-o` argument, you could just call `save_image()` if it's set.